metron-interface/metron-rest/src/main/java/org/apache/metron/rest/controller/MetaAlertController.java

```diff
@@ -60,5 +63,37 @@
   ) throws RestException {
     return new ResponseEntity<>(metaAlertService.create(createRequest), HttpStatus.OK);
   }
+
+  @ApiOperation(value = "Create a meta alert")
```

Comment:
The descriptions and so on need to be updated on all of these.  I know you're working on documentation, but I wanted to call it out so it doesn't slip through.